setup.py

```diff
@@ -149,6 +149,7 @@ def run_tests(self):
         versioneer.get_cmdclass(),
         {"test": NoseTestCommand}
     ]], [])),
+    conda_features=[],
```

Comment:
Adding this workaround seems to work.

Also changing the default of `conda_features` in `bdist_conda` to `[]` works just as well. Have added a PR ( https://github.com/conda/conda-build/pull/1634 ) upstream to change the default value of `conda_features` to avoid these issues.